ui/balances_table.ts

```diff
@@ -89,11 +96,11 @@ function total_table_recreate() {
     update_last_balance_save();
 }
 
-function total_table_add_balances(location, query_result) {
+export function total_table_add_balances(location: string, query_result: any) {
```

Comment:
Here `query_result` should be an object. It's value is something like `{'USD': '100', 'BTC': '1'}`